newsfragments/2886.fixed.md

```diff
@@ -0,0 +1 @@
+Send errors returned by `__releasebuffer__` to `sys.unraisablehook` rather than causing `SystemError`.
```

Comment:
This does also apply the same change for our implementations of dealloc, does it not?

Reply:
Yes, though `dealloc` is internal and we don't currently return any errors from it, so I don't think this would change any user-facing behaviour.